packages/graphback-cli/src/templates/resources/models/Tasks.graphql

```diff
@@ -3,11 +3,16 @@ type Task {
   version: Int
   title: String!
   description: String!
-  status: TaskStatus
-  createdBy: User
+  status: String!
+  creationMetadata: CreationMetadata
   assignedTo: User
 }
 
+type CreationMetadata {
```

Comment:
@ankitjena - good example of the model that needs to be `@genignore`'d. 
We use it for relationships only and do not need full crud on this. 
See #187